### DISCUSSION TITLE
Images Are Gone

### DISCUSSION THREAD

**Discussion by infamousjae** [iterate-ch/cyberduck#15201] [Q&A]
Any help on this would be greatly appreciated. We logged into our public cyberduck S3 domain and noticed that ALL of our images are gone. This is two years worth of article images. How do we get these back?

**Comment by tatze96** [comment]
Cyberduck can't delete files by itself.

Please check within the server's web interface if the files are still present to rule out a listing issue.
Also, verify if the files could have been removed through any present lifecycle settings.
